cogboard-webapp/src/components/Widget/index.js

```diff
@@ -135,7 +135,7 @@ const Widget = ({ id, index }) => {
         isOver={isOver}
         ref={ref}
       >
-        {(isAuthenticated || title !== '') && (
+        {(isAuthenticated || widgetStatus !== 'NONE' || title !== '') && (
```

Comment:
is this really Ok there is no icon for 'NONE' status registered so this is probably redundant

Reply:
It is necessary if we want to show status icon when there is no title and user is logged out. Currently if widget has for example success status it is not shown when there is not title.